docs/decisions/006-default-style.md

```diff
@@ -0,0 +1,101 @@
+# Apply ECL 3 styling to default HTML tags
+
+| Status        | proposed                                              |
+| ------------- | ----------------------------------------------------- |
+| **Proposed**  | 12/10/2020                                            |
+| **Accepted**  | --/--/----                                            |
+| **Driver**    | @emeryro                                              |
+| **Approver**  |                                                       |
+| **Consulted** | @kalinchernev, @planctus                              |
+| **Informed**  | [@team](https://github.com/orgs/ec-europa/teams/inno) |
+
+## Decision
+
+[To be added]
+
+## Context
+
+- Styling of default HTML tags (without CSS class) has been a redundant request from some ECL users
+- In ECL 2, package `ecl-editor` was provided as a workaround. It worked quite well, but ended up raising some CSS conflict issues like [FRONT-1697](https://citnet.tech.ec.europa.eu/CITnet/jira/browse/FRONT-1697)
+- In ECL 3 we shoud come with a more robust way to handle this, without disturbing the recommanded way to use ECL (by adding CSS classes)
+- We have no control over the final markup of pages. So we can't rely on any markup based selectors (like `>`, `+`, ...)
+
+## Requirements
+
+- This default styling should be made available as an optional CSS, or in an optional preset
```

Comment:
These requirements are not the ones we always considered, i'd say: accepting or even embracing the idea of styling basic and limited html tags never meant we would support "every html tag".
We already defined the list of these supported tags, which are only `headings, global font definition, lists, tables`.

I have the impression we confuse the ecl-editor with this, because in the past it could have been used also as a workaround for styling un-themed markup, but ideally in v3 the editor would only have to ensure the "correct" rendering of a component markup once it is wrapped in some other html (.ecl-editor in particular), while the definition of basic styles for headings would let them, for instance, define an h2 inside the editor without having to specify a class.

I don't even see the reason why we would again create a "separated" something for this, like for saying we don't want it and we only do it for who requested this and it will be not be part of the standard css defined by the library, we don't have "presets" in v3 in terms of differentiated final output, we would reintroduce them just for this.

Anyway, overall, my preference would be for "ckeditor plugins", theming the markup as requested in ECL, the info website has some of these widgets (like headings for instance) since years, also the table widget is customised in info to output the desired markup, it would super easy to add a class to every block of text (paragraphs), i wonder how can this still be an issue for the d8 version of the platform. 
Surely we are not enthusiast about doing something that we feel might not be needed, but again, instead of making a big fuss around this, we have to simply set some global style, not reinventing ECL without markup, imho.

Reply:
There are indeed confusion here.
Maybe I missed it, but I don't know where your list of tags to be styled comes from (I don't remember seeing this before).
Also, just to be clear, there is no intention to have something like ecl-editor in ECL 3. That's why these global styling are considered a more robust way to replace it

Reply:
We explicitely mentioned the tags we would style in many occasions, because we knew we had to limit this to a reasonable amount, if you need to find this back we probably have this list in one of the multiple places where we dislocated our conversations, documents, etc..but to me, it's not a matter of what we decided before, is that this is still the scope of this operation, not a different one, styling basic html tags, surely not making ECL work with whatever markup.

Reply:
I think Davide's list is pretty much ok `headings`, `global font definition` (paragraphs, links, blockquotes), `lists`, `tables` maybe a default primary for `button`. however I don't think a ckeditor plugin is the way to go. There are other markup outputting from Drupal without going through ckeditor that must be styled, such as a simple link that will need to be preprocessed to add the class and get the right style. Adding classes and changing markup to fields saved in database is not a reliable solution in my opinion, ckeditor already leaves too much flexibility in this direction in my opinion.